src/aggregator-ruleng/WorkItemRelationWrapperCollection.cs

```diff
@@ -20,7 +20,8 @@ internal WorkItemRelationWrapperCollection(WorkItemWrapper workItem, IList<WorkI
             _pivotWorkItem = workItem;
             _original = relations == null
                 ? new List<WorkItemRelationWrapper>()
-                : relations.Select(relation => new WorkItemRelationWrapper(relation))
+                : relations.Where(relation => relation.Rel.StartsWith("System.LinkType"))
```

Comment:
This will filter too much, as not all relations are start with System.LinkType, see https://docs.microsoft.com/en-us/azure/devops/boards/queries/link-type-reference?view=azure-devops

For this filter we need a little bit more testing.

Can you describe a little bit more what the issue is? As soon as you have an attachment linked in the workitem, it cannot be handled?

Reply:
Sorry, too fast !
Indeed, when the relation is an attachment, the last segment in url relation isn't an id but a UID, which causes this error : "Input string was not in a correct format".